core/src/avm2/globals/flash/desktop/native_application.rs

```diff
@@ -0,0 +1,17 @@
+use crate::avm2::{Activation, ArrayObject, ArrayStorage, Error, Object, Value};
+use crate::string::AvmString;
+
+pub fn get_argv_internal<'gc>(
```

Comment:
I am not a fan of the c-ism `argv`. Maybe `args` like Rust or `arguments` like Flash?

Reply:
I agree, `arguments` is better, I will change that.